account_voucher_supplier_invoice_number/voucher.py

```diff
@@ -40,10 +40,10 @@ def _get_supplier_invoice_number(self, cr, uid, ids, name, args, context=None):
     
     _columns = {
         'supplier_invoice_number': fields.function(_get_supplier_invoice_number,
-            type='char', size=64, string="Supplier Invoice Number"),
+            type='char', size=64, string=_("Supplier Invoice Number")),
         }
 
-class voucher(orm.Model):
+class voucher(models.Model):
     _inherit = 'account.voucher'
     
     def recompute_voucher_lines(self, cr, uid, ids, partner_id, journal_id, price,
```

Comment:
You are mixing new and old API. If you do all in new API, you won't find these problems.